integration-tests/README.md

```diff
@@ -132,7 +132,7 @@ Other custom configuration settings available:
     is specified
   * `-D coreapi_server_image=bayesian/bayesian-api`
     (optional, default is `bayesian/bayesian-api`) - name of Bayesian core API server image
-  * `-D cucos_worker_image=bayesian/cucos-worker` (optional, default is `bayesian/cucos-worker`) - name of Bayesian
+  * `-D f8a_worker_image=bayesian/cucos-worker` (optional, default is `bayesian/cucos-worker`) - name of Bayesian
```

Comment:
The [code](https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/features/environment.py#L264) actually reads from `coreapi_worker_image` configuration setting, so please rename to that one.